index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bierproductie: Info Pages</title>
+    <link rel="stylesheet" href="styles/main.css"/>
+    <script src="scripts/script.js" defer></script>
+    <script src="scripts/maintain.js" defer></script>
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+  </head>
+  <body>
+
+    <!-- Side navigation -->
+    <div class="sidenav">
+      <span class="iconify marked-page" onclick="location.href='/';" data-icon="cil:touch-app" data-inline="false">
+      </span>
+      <span class="iconify" onclick="location.href='/settings.html';" data-icon="carbon:settings" data-inline="false">
+      </span>
+    </div>
+
+    <!-- topbar -->
+    <header>       
+      <h2>Bierproductie</h2>
+      <div class="batch" style="background-color: transparent;">
+        <p id="batch">BATCH</p>
```

Comment:
I think we need some space between the "BATCH" and the batch numbers. Also, it would be great you could press the number as well to get the dropdown menu and not just the small arrow. (which is going to be hard to press on a touch screen).

![image](https://user-images.githubusercontent.com/55458479/99437148-5e1c5f80-2912-11eb-98ae-1380b18c258d.png)

Reply:
yes exactly 
yes it will make sense you could click on the number aswell. 
- More space
- Pres number should show the list